source/bdDetect.py

```diff
@@ -228,42 +261,27 @@ def _get_usbDevices(self) -> typing.List[typing.Dict]:
 	def _get_hidDevices(self) -> typing.List[typing.Dict]:
 		return list(hwPortUtils.listHidDevices(onlyAvailable=True))
 
-#: The single instance of the device info fetcher.
-#: @type: L{_DeviceInfoFetcher}
-deviceInfoFetcher = _DeviceInfoFetcher()
+
+deviceInfoFetcher: _DeviceInfoFetcher
+
 
 class Detector(object):

```

Comment:
since we are "breaking" the API here, let's explicitly mark this as private.

```suggestion
class _Detector:
```